pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -95,22 +96,70 @@ func (ic *ifConfigurator) moveVFtoContainerNS(vfNetDevice string, containerID st
 	hostIface := result.Interfaces[0]
 	containerIface := result.Interfaces[1]
 
+	link, err := ic.netlink.LinkByName(vfNetDevice)
+	klog.V(2).Infof("Get link of vfNetDevice: %s, link: %+v", vfNetDevice, link)
+	if err != nil {
+		return fmt.Errorf("error getting VF link: %v", err)
+	}
+
+	// Rename the device in a temp NS to avoid race condition.
```

Comment:
This is how host device handles renaming?

Reply:
Yes, it uses a temporary namespace to do rename.

Reply:
You should probably include a link to the host-device plugin code for future reference